web/themes/custom/cloudy/templates/views/views-view-fields--field-set--block-1.html.twig

```diff
@@ -0,0 +1,72 @@
+{#
+/**
+ * @file
+ * Theme override to display all the fields in a row.
```

Comment:
I'm not certain what this view template is overriding in the current implementation. All the variables seem to be date related. If we are going to keep this, we should at least update the notes and perhaps rename the block_1 machine name to something with a bit more meaning. 🤔

Reply:
I'm removing this view, added by mistake.